docker/ebpp/config.toml

```diff
@@ -6,14 +6,13 @@ bind_address = "0.0.0.0:3338"
 
 grpc_address = "0.0.0.0:9090"
 electrum_url = "electrs:50001"
-refresh_interval=1000
+refresh_interval=30
```

Comment:
nitpick, but I like it if these kinds of config values have the unit attached to them, e.g. `refresh_interval_seconds`